data/json/snippets.json

```diff
@@ -1464,7 +1464,10 @@
       "GLAMOPOLITAN!  We've got ALL the latest tips!  Whether you want to know what the elite are eating, wearing or discussing, Glamopolitan is YOUR magazine!  So pick up a copy today and \"Sizzle Like A Star\"!",
       "POPULAR MECHANICS: People say mechanics is boring.  We say, Prove them Wrong!  We've got all the articles that make it interesting to talk about, so you can \"Make Mechanics Popular\"!",
       "CRAFTY CRAFTERS QUARTERLY: Macaroni isn't just for eating anymore!  Learn how to make jewelery and art from it as well!  We also discuss the correct way to use superglue without gluing your hands together!",
-      "UNDERGROUND BUNKER?  Sources in the government suggest that a underground bunker is being built in a secret location in case of enemy attack.  We were unable to get the exact location as our sources mysteriously disappeared shortly after revealing this information."
+      "UNDERGROUND BUNKER?  Sources in the government suggest that an underground bunker is being built in a secret location in case of enemy attack.  We were unable to get the exact location as our sources mysteriously disappeared shortly after revealing this information.",
+      "STUDENT MISSING: A high school student vanished yesterday evening in the forest near Wayland.  The 17-year-old international student from China (who adopted the anglicized nickname \"Brett\" due to Americans' difficulty pronouncing his given name) was last seen with his friends in the camp.  \"Brett said that he was gonna get some firewood but he never came back,\" said his classmate, Jianxiang Wang.  The search is underway.",
+      "STILL SEARCHING: The search for Brett, the high school student went missing three days ago, is still ongoing.  The County Search & Rescue was brought in, however, the missing midfielder for his school soccer team had still not been located as of writing this report.  \"He could have played in the match yesterday,\" says sorrowfully his teammate, \"[..] we've never stopped praying.\"",
+      "RUMORS DENIED: Rumors had been swirling since mid-July about the military conducting secret teleportation experiments in an underground research facility near Wayland.  \"I have to clarify is that we'd never done such research,\" says the officer on the press conference, \"Teleportation only exists in sci-fi films.\""
```

Comment:
I'm definitely going overboard editing an easter egg in a snippet but whatever: this snippet only mentions a press conference indirectly ("says the officer on the press conference") - it'd be better to mention that outright. Maybe something like this:

> "RUMORS DENIED: Allegations that the military had been conducting teleportation experiments in secret just outside Wayland were put to rest during a press conference earlier today.  \"I have to clarify that not only have we never done such research,\" said the officer on the press conference, \"teleportation only exists in sci-fi films.\""